app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.nil?
+      render json: {message: "User Not Found"}, status: :not_found
+    else
+      @user.update(user_params)
```

Comment:
should be in an `if`, otherwise if it fails we wouldn't know in our frontend :bomb: